argilla-server/src/argilla_server/validators/records.py

```diff
@@ -126,6 +128,38 @@ def _validate_image_field(cls, field_name: str, field_value: Union[str, None]) -
         else:
             raise UnprocessableEntityError(f"image field {field_name!r} has an invalid URL value")
 
+    @classmethod
+    def _validate_chat_fields(cls, dataset: Dataset, fields: Dict[str, str]) -> None:
+        for field in filter(lambda field: field.is_chat, dataset.fields):
+            cls._validate_chat_field(field.name, fields.get(field.name))
+
+    @classmethod
+    def _validate_chat_field(cls, field_name: str, field_value: Union[str, None]) -> None:
+        if field_value is None:
+            return
+
+        if len(field_value) > CHAT_FIELD_MAX_LENGTH:
+            raise UnprocessableEntityError(
+                f"chat field {field_name!r} value is exceeding the maximum length of {CHAT_FIELD_MAX_LENGTH} characters"
+            )
```

Comment:
I don't get this. Is not the chat field value a list of objects? Then should the message be changed?

Reply:
Yes. This message is wrong. It should be 'max messages'. I will fix when we resolve the pydantic decision.